diff --git a/node_modules/@arch-ui/select/dist/select.esm.js b/node_modules/@arch-ui/select/dist/select.esm.js
index 28a25c6..e6f6c6b 100644
--- a/node_modules/@arch-ui/select/dist/select.esm.js
+++ b/node_modules/@arch-ui/select/dist/select.esm.js
@@ -41,6 +41,7 @@ const selectStyles = {
   control: (provided, {
     isFocused
   }) => {
+    
     const focusStyles = isFocused ? {
       borderColor: colors.primary,
       boxShadow: `inset 0 1px 1px rgba(0, 0, 0, 0.075),
@@ -69,6 +70,7 @@ const selectStyles = {
     isFocused,
     isSelected
   }) => {
+
     let bg = 'inherit';
     if (isFocused) bg = colors.B.L90;
     if (isSelected) bg = colors.primary;
@@ -119,7 +121,7 @@ const Select = (_ref) => {
     styles
   } = _ref,
       props = _objectWithoutProperties(_ref, ["isAsync", "isCreatable", "innerRef", "styles"]);
-
+console.log(props)
   const ReactSelect = getSelectVariant({
     isAsync,
     isCreatable
